src/test/java/bio/terra/janitor/db/DatabaseTestUtils.java

```diff
@@ -0,0 +1,20 @@
+package bio.terra.janitor.db;
+
+import bio.terra.janitor.app.configuration.JanitorJdbcConfiguration;
+import bio.terra.janitor.service.migirate.MigrateService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/** Test utils for databases. */
+@Component
+public class DatabaseTestUtils {
```

Comment:
why we need this?  DB will get dropped and recreate by this:
https://github.com/DataBiosphere/terra-resource-janitor/blob/master/src/test/resources/application.properties

Reply:
That's between runs of the entire test suite, not per test.

Reply:
Ah, we have the same pain, then I end up using random UUID as the project name instead of a fixed value. 
Actually the DB is easily get messed up, there should be some way to tell Spring reset per test run. 
one think I just find out is to use this:
`@DirtiesContext(classMode = ClassMode.AFTER_EACH_TEST_METHOD)` But I haven't try it yet.
https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/test/annotation/DirtiesContext.html
Do you want to try this out?

Reply:
Thanks that works.
We might get to the point where it's painfully slow to recreate the entire app each time, but let's deal with that if/when we get there.

Reply:
ah, okay, drop and creating db also take some time, but I agree restart server would take bit longer than that.